makefiles/tools/serial.inc.mk

```diff
@@ -21,4 +21,9 @@ else ifeq ($(RIOT_TERMINAL),socat)
 else ifeq ($(RIOT_TERMINAL),picocom)
   TERMPROG  ?= picocom
   TERMFLAGS ?= --nolock --imap lfcrlf --baud "$(BAUD)" "$(PORT)"
+else ifeq ($(RIOT_TERMINAL),miniterm)
+  TERMPROG  ?= miniterm.py
+  # The RIOT shell will still transmit back a CRLF, but at least with --eol LF
```

Comment:
> The RIOT shell will still transmit back a CRLF

really?

Reply:
I don't understand neither the issue nor the comment. RIOT *should* send LF on every "\n", which the terminal is supposed to turn into "\r\n" before sending to the console, so "--eol LF" is definitely correct.

Reply:
> I don't understand neither the issue nor the comment.

This is taken as-is from the original PR. Do you have a better suggestion for the comment ?

Reply:
Ah, what's happening is that miniterm sends "\r\n" on enter, and the RIOT shell accepts both as newline.